### ISSUE TITLE
JSON error

### ISSUE THREAD

**Issue by Nevos08** [snakewiz/fivem-appearance#4]
This is the console output:
`Uncaught (in promise) SyntaxError: Unexpected end of JSON input (@fivem-appearance/ui/build/static/js/main.53ee5267.chunk.js:1)`

**Comment by Davenport-Physics** [comment]
I had the same issue since v1.1

The only workaround I've found, is to restart the resource while the server is up :shrug:

**Comment by Nevos08** [comment]
Thanks will try :D

**Comment by snakewiz** [comment]
Hi, thanks for the feedback. I need the actual steps to reproduce this issue before I can fix it, if you could provide me that, it would be of great help.

**Comment by Nevos08** [comment]
I just downloaded the script from master (no release!), removed bodyArmor from the menu and then compiled it.

**Comment by snakewiz** [comment]
> I just downloaded the script from master (no release!), removed bodyArmor from the menu and then compiled it.

I mean in-game steps.

**Comment by Nevos08** [comment]
The error came when the script started. it didnt work to open the menu via exports

**Comment by computergeek3k** [comment]
can confirm fresh download of the precompiled resource, I dropped it in game start the resource and once in game hit f8 which displayed the above error

**Comment by Nevos08** [comment]
Any fix atm?

**Comment by morpheause** [comment]
![image](https://user-images.githubusercontent.com/54166713/115939821-12b14300-a4a8-11eb-83fe-df98b1d4e2e1.png)

**Comment by snakewiz** [comment]
Sorry for the delay, the last few weeks have been very busy. This error is caused by not loading the UI files when fivem-appearance:customization conVar isn't set to 1.

This shouldn't cause any problems if you're just trying to use the appearance exports.

If you want to use the customization feature you need to add the following to your server .cfg file.

```cfg
setr fivem-appearance:customization 1
```

**Comment by Nevos08** [comment]
I think I did that. I will retry with the newest release

**Comment by Nevos08** [comment]
Im getting an error now when building the project. Its something with prettier.

Something like this:
![Screenshot 2021-04-28 175525](https://user-images.githubusercontent.com/54755752/116434688-ecdec200-a84a-11eb-9313-92030bec183a.png)

**Comment by RockySouthpaw** [comment]
> Sorry for the delay, the last few weeks have been very busy. This error is caused by not loading the UI files when fivem-appearance:customization conVar isn't set to 1.
> 
> This shouldn't cause any problems if you're just trying to use the appearance exports.
> 
> If you want to use the customization feature you need to add the following to your server .cfg file.
> 
> ```haproxy-config
> setr fivem-appearance:customization 1
> ```

Figured I would provide some additional information as I also have this issue.

`Server.cfg` is as such:
```cfg
setr fivem-appearance:customization 1
setr fivem-appearance:locale "en"
ensure fivem-appearance
```
Release Version: `1.12` (Latest from release tab.)

With nothing triggering the export what so ever, I get the same error as mentioned previously:
![image](https://user-images.githubusercontent.com/55056068/117118930-60a73000-ad5f-11eb-9110-3971536dd62d.png)
This requires the resource to be restarted in order for it to function again.

**Comment by snakewiz** [comment]
Fixed on 68b60e0b74af24bcb5abad47469496763d2fd4f6, I'm sorry that this took so long. The UI page was getting loaded even if the convar was set to 0 because 0 is not a falsy value in Lua. Thanks to everyone that reported the issue.
